src/sentry/charts/chartcuterie.py

```diff
@@ -0,0 +1,81 @@
+from io import BytesIO
+from typing import Any, Union
+from urllib.parse import urljoin
+from uuid import uuid4
+
+import sentry_sdk
+from django.conf import settings
+
+from sentry import options
+from sentry.exceptions import InvalidConfiguration
+from sentry.models.file import get_storage
+from sentry.net.http import Session
+from sentry.utils.http import absolute_uri
+
+from .base import ChartRenderer, logger
+from .types import ChartType
+
+
+class Chartcuterie(ChartRenderer):
+    """
+    The Chartcuterie service is responsible for converting series data into a
+    chart of the data as an image.
+
+    This uses the external Chartcuterie API to produce charts
+    """
+
+    @property
+    def service_url(self) -> str:
+        return options.get("chart-rendering.chartcuterie", {}).get("url")
+
+    def validate(self):
```

Comment:
I'm actually not sure when `validate` is run, but I think it might be a service initialization thing, so we don't want to validate anything if the service is disabled.